test/integration/vite/src/main.tsx

```diff
@@ -35,64 +43,72 @@ function Root() {
     const location = useLocation();
 
     return (
-        <div style={{ "height": "100vh", "display": "flex", "flexDirection": "column" }}>
-            <Header
-                brandTop={brandTop}
-                serviceTitle="Nom du site / service"
-                homeLinkProps={homeLinkProps}
-                quickAccessItems={[headerFooterDisplayItem]}
-                navigation={[
-                    {
-                        "text": "Home",
-                        "linkProps": {
-                            "to": "/"
+        <GdprStoreProvider>
+            <ConsentBanner gdprPageLink="/mui" siteName='Next Test App' services={[
```

Comment:
If we do that we're breaking the client side routing.  
We can't assume what routing library is used (react-router, TanStack router, Next, vanilla, type-route,...) the user provide the link component and overwrite the type of the link props when setting up the dsfr.  

https://react-dsfr.etalab.studio/routing

Reply:
I don't understand what you mean here or why the client routing would be broken ; do you think it's better for the provider to also take a "Link" component as prop instead of using one from `getLink`?

Reply:
Or maybe event directly registered Link props ?